readthedocs/projects/tasks.py

```diff
@@ -984,3 +984,8 @@ def clear_html_artifacts(version):
     if isinstance(version, int):
         version = Version.objects.get(pk=version)
     remove_dir(version.project.rtd_build_path(version=version.slug))
+
+
+@task()
+def bulk_delete_projects(projects_id):
+    Project.objects.filter(id__in=projects_id).delete()
```

Comment:
Any reason this needs to be executed via a task?

Reply:
I was hoping to delete the project directory with the project removal. So was thinking to do it with task. I think we can exclude it from the task